ui/src/main/java/com/doordeck/sdk/ui/verify/VerifyDeviceActivity.kt

```diff
@@ -206,7 +207,12 @@ internal class VerifyDeviceActivity : BaseActivity(), VerifyDeviceView {
 
     // the code is valid, close the with and come back on the previous screen : The UnlockActivity
     override fun succeed() {
-        finish()
+        if (shouldUnlockAfterSuccessVerifying) {
+            finish()
+            Doordeck.showUnlock(this)
+        } else {
+            finish()
+        }
```

Comment:
```suggestion
        finish()
        if (shouldUnlockAfterSuccessVerifying) {
            Doordeck.showUnlock(this)
        }
```

Reply:
Pushed, thanks.
That's me testing order stuff :D and committing what worked